docs/vsphere_ci/scripts/powershell.ps1

```diff
@@ -0,0 +1,21 @@
+Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
+$firewallRuleName = "ContainerLogsPort"
+$containerLogsPort = "10250"
+New-NetFirewallRule -DisplayName $firewallRuleName -Direction Inbound -Action Allow -Protocol TCP -LocalPort $containerLogsPort -EdgeTraversalPolicy Allow
+Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
+Install-Module -Force OpenSSHUtils
```

Comment:
@sebsoto @ravisantoshgudimetla I should have caught this before. This will soon stop working and we should be doing what we have in the [user data secret](https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/secrets/secrets.go#L61).

cc: @christianh814